docs/handbook/image-file-formats.rst

```diff
@@ -525,13 +525,18 @@ For more information about the SPIDER image processing package, see the
 TIFF
 ^^^^
 
-PIL reads and writes TIFF files. It can read both striped and tiled images,
-pixel and plane interleaved multi-band images, and either uncompressed, or
-Packbits, LZW, or JPEG compressed images.
+Pillow reads and writes TIFF files. It can read both striped and tiled
+images, pixel and plane interleaved multi-band images. If you have
+libtiff and its headers installed, PIL can read and write many kinds
+of compressed TIFF files. If not, PIL will only read and write
+uncompressed files.
 
-If you have libtiff and its headers installed, PIL can read and write many more
-kinds of compressed TIFF files. If not, PIL will always write uncompressed
-files.
+.. note::
+
+    Beginning in version 4.4.0, Pillow requires libtiff to read or
```

Comment:
version 5.0

Reply:
We're going to have to do a blanket 4.4/5.0 replace.